### ISSUE TITLE
Multiple .Filter

### ISSUE THREAD

**Issue by ChristofferBergkvist** [object/Simple.OData.Client#239]
Chaining of multiple .Filters does not work.

In my case I'm trying to combine the following:
var data = await context.For<T>().Filter(myString).Filter(myExpression).FindEntriesAsync()

where myString is a string an myExpression is Expression<Func<T, bool>>

They work great one by one though.

Is there a workaround so I can combine them into one .Filter maybe?

**Comment by object** [comment]
Hi,
Actually you can build expressions from multiple filters. Have a look at the tests at the bottom of the file:

https://github.com/object/Simple.OData.Client/blob/master/Simple.OData.Client.Tests.Core/TypedExpressionsTests.cs

**Comment by ChristofferBergkvist** [comment]
Yes I know that. But is it possible to combine filters when one filter parameter is a string?
`var data = await context.For("MyClass").Filter("myString").Filter(myExpression).FindEntriesAsync()`

**Comment by object** [comment]
Oh I see, you want to combine expressions and strings. No it's not possible and I don't think will ever be. When you use strings as filters you take full control of the filter semantics, the library simply passes the filter to the server. In case of expressions the library is in charge. What you can do is to obtain the command string using GetCommandTextAsync call and then rewrite it and add more filters.
